docs/pages/connect-your-client/tsh.mdx

```diff
@@ -80,10 +80,12 @@ to call [`tsh login`](../reference/cli.mdx#tsh-login) in the beginning.
 
 ## Installing tsh
 
-Follow [these install instructions](../installation.mdx) to obtain the `tsh`
-binary. Ideally, install `tsh` of the same version as the version used in your
+Follow install instructions to obtain the `tsh` binary. Ideally, install
```

Comment:
What does "Ideally" mean here? If this is intended to imply that sometimes it's okay for `tsh` to not have the same version as the one used in your Teleport cluster, I think we should be explicit about when that is.

Reply:
Please see update.